metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -18,6 +18,21 @@ class ImportMissingSymbol(compilers: Compilers) extends CodeAction {
       token: CancelToken
   )(implicit ec: ExecutionContext): Future[Seq[l.CodeAction]] = {
 
+    val uri = params.getTextDocument().getUri()
+
+    def joinActionEdits(actions: Seq[l.CodeAction]) = {
+      actions
+        .flatMap(_.getEdit().getChanges().get(uri).asScala)
+        .distinct
+        .groupBy(_.getRange())
+        .values
+        .map(_.sortBy(_.getNewText()).reduceLeft { (l, r) =>
+          l.setNewText(l.getNewText() + r.getNewText() replace ("\n\n", "\n"))
```

Comment:
Super nitpicky, but just for consistency, can we avoid infix notation here? I know it was just copied from below.
```suggestion
          l.setNewText(l.getNewText() + r.getNewText().replace("\n\n", "\n"))
```

Reply:
fixed!

Reply:
Although funnily enough adding a `.` make the replace be applied only on `r`. Without the dot it was applied on the sum of the two strings.

Reply:
Ah, yea I guess that makes sense though. Good catch.